exercises/alphametics/example.rb

```diff
@@ -0,0 +1,41 @@
+class Alphametics
+  VERSION = 1
+
+  def solve(puzzle)
+    letters = Hash.new(0)
+    puzzle = format_to_ruby(puzzle)
```

Comment:
I think it is fine either way.  It doesn't indicate that it should be evaluated by Ruby itself.  For instance, there is no `instance_eval(format_to_ruby(puzzle))` kind of thing being done.

I think it has some value to format it to a 'normalized ruby expression' though, it gives a little extra something in the exercise.

The method name `as_ruby_expression` may be good?  I don't know, naming is hard.